rrfs-test/testinput_expr/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -99,134 +100,86 @@ cost function:
+         - name: airTemperature@GsiUseFlag 
+         minvalue: 0.5 
+         maxvalue: 1.5 
+         action:
+            name: accept 
 
-         # Reject airTemperature with ObsType=195
-         - filter: Perform Action
-           filter variables:
-           - name: airTemperature
-           where:
-           - variable: ObsType/airTemperature
-             is_in: 195
-           action:
-             name: reject
-
-         - filter: Bounds Check
-           filter variables:
-           - name: airTemperature
-           minvalue: 200
-           maxvalue: 400
+     - obs space:
+         name: sonde  
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: Data/obs/sondes_tsen_obs_2022052619.nc4
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: rass_tsen_obs_2022052619_hofx.nc4
+         simulated variables: [airTemperature]
+         observed variables: [airTemperature]
+       obs operator:
+         name: VertInterp
+       obs error:
+         covariance model: diagonal
+       obs filters:
+       - filter: Bounds Check
+         filter variables:
+         - name: airTemperature@GsiUseFlag 
+         minvalue: 0.5 
+         maxvalue: 1.5 
+         action:
+            name: accept 
 
-         - filter: Background Check
-           filter variables:
-           - name: airTemperature
-           threshold: 4.0
-           where:
-           - variable: ObsType/airTemperature
-             is_in: 188
-           action:
-             name: reject
 
-        # Print filter data
-         - filter: Print Filter Data
-           message: Printing filter data
-           summary: true
-           variables:
-           - variable: MetaData/latitude
-           - variable: MetaData/longitude
-           - variable: MetaData/pressure
-           - variable: ObsType/airTemperature
-           - variable: ObsValue/airTemperature
-           - variable: QCflagsData/airTemperature
+     - obs space:
+         name: sonde
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: Data/obs/sondes_uv_obs_2022052619.nc4
+         simulated variables: [windEastward, windNorthward]
+       obs operator:
+         name: VertInterp
+         vertical coordinate: air_pressure
+         observation vertical coordinate: pressure
+         interpolation method: log-linear
+       obs error:
+         covariance model: diagonal ufo
+       obs filters:
+       - filter: PreQC
+         maxvalue: 3
+       - filter: Background Check
+         filter variables:
+         - name: windEastward
+         - name: windNorthward
+         threshold: 6.0
+       monitoring only: true
 
-         - filter: GOMsaver
-           filename: ./msonet_geovals_rrfs_2022052619.nc4
 
 variational:
   minimizer:
-    algorithm: DRPCG    # Derber-Rosati Conjugate Gradients
+    algorithm: DRPCG
   iterations:
-  - ninner: 50
+  - ninner: 5
```

Comment:
for real data ,  ninner=5 is too small?

Reply:
This PR is only updating the yamls to use more data for the MPAS-JEDI tests. These fv3jedi yamls use smaller sets of obs (~800 total obs). I don't mind updating `ninner` to 50 for the fv3jedi case but those files were not created in this PR (only copied between `testinput` and testinput_expr`).

Reply:
Yes, there are multiple copies of the fv3jedi yamls. They are located in both `rrfs-test/testinput` and `rrfs-test/testinput_expr`. I should probably remove them from `testinput_expr` since that directory was only meant to save the older yamls that other developers might still be using (that were previously used for ctests). The new ctest yamls will be located in `rrfs-test/testinput`.